api/Document.json

```diff
@@ -6084,6 +6188,726 @@
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": "37"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "pointerleave_event": {
+        "__compat": {
+          "description": "<code>pointerleave</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/pointerleave_event",
+          "support": {
+            "chrome": {
+              "version_added": "35"
+            },
+            "chrome_android": {
+              "version_added": "35"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "59"
+              },
+              {
+                "version_added": "29",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.w3c_pointer_events.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              }
+            ],
+            "firefox_android": {
+              "version_added": "29",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.w3c_pointer_events.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "ie": [
+              {
+                "version_added": "11"
+              },
+              {
+                "version_added": "10",
+                "alternative_name": "mspointerleave"
+              }
+            ],
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": "37"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "pointerlockchange_event": {
+        "__compat": {
+          "description": "<code>pointerlockchange</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/pointerlockchange_event",
+          "support": {
+            "chrome": {
+              "version_added": null
```

Comment:
This was unprefixed in Chrome 45 for desktop, Android, and webview. https://storage.googleapis.com/chromium-find-releases-static/6bf.html#6bf459eeba20b0c96a2576be823b7066cd207f05

The prefixed interfaces were also removed in 45. https://storage.googleapis.com/chromium-find-releases-static/c1f.html#c1f0e8bca46cddc3b448571bcc0752e1cf168d73

The prefixed versions were committed on June 7, 2012. Our release finding tool doesn't have a version number for this, but the date puts it in Chrome 22. https://chromium.googlesource.com/chromium/src/+/5d45e2512bf4ca0d6a47f96d0f18e0f009ba5043

Reply:
Thanks @jpmedley !

As I understand it then, a structure like this ought to cover it:

```
[
              {
                "version_added": "45"
              },
              {
                "version_added": "22",
                "version_removed": "45",
                "prefix": "webkit"
              }
            ],
```
?

Given your comment in here: https://github.com/mdn/browser-compat-data/pull/3469#discussion_r258205578 that "Chrome Android skipped versions 19 through 24", I've used "25" not "22" for that browser. Also neither "22" nor "25" are valid for webview, so I have just set that one to `true`.

I assume we should do this for `Element` as well?